inputFiles/thermoPoromechanicsFractures/ThermoPoroElastic_efem-edfm_eggModel_small.xml

```diff
@@ -219,7 +219,8 @@
 
     <SolidInternalEnergy
       name="rockInternalEnergy"
-      volumetricHeatCapacity="1.672e5"
+      referenceVolumetricHeatCapacity="1.672e5"
+      dVolumetricHeatCapacity_dTemperature="1e6"
```

Comment:
dVolumetricHeatCapacity_dTemperature for this case should be zero (default value)?
```suggestion

```

Reply:
Yes, done